xbmc/utils/ContentUtils.cpp

```diff
@@ -9,8 +9,15 @@
 #include "ContentUtils.h"
 
 #include "FileItem.h"
+#include "ServiceBroker.h"
```

Comment:
When this file was initially created with was not supposed to have GUI stuff in it. With this change you are binding GUI to any file that depends on it...

Reply:
> > When this file was initially created with was not supposed to have GUI stuff in it.
> 
> Really? Reading method names like "HasPreferredArtType" provided by ContentUtils.h, I think this is GUI stuff contained here. I might be wrong, though.
> 

In my opinion UI != GUI, `HasPreferredArtType` still has value even if Kodi is running without it's GUI as you can use that information to present art on a webinterface. If I remember correctly when I created this class it's already used somewhere in the json interface (maybe indirectly).

Maybe derive from this class in another one where you can shuffle the actual GUI stuff? Other ideas are welcome.

Reply:
Well, I will create utils/guilib/GUIContentUtils.(cpp|h)

Reply:
done